client/ayon_core/hosts/houdini/plugins/publish/collect_local_render_instances.py

```diff
@@ -64,6 +64,8 @@ def process(self, instance):
         expectedFiles = next(iter(instance.data["expectedFiles"]), {})
 
         product_type = "render"  # is always render
+        # TODO: Match the value of product_group with the value of
+        #        group_name in farm pyblish functions.
         product_group = get_product_name(
             context.data["projectName"],
             context.data["taskEntity"]["name"],
```

Comment:
for reference, 
I gave this PR a test. and was going to mention an issue with the product group name in local Houdini render.
Then, I figured out there was a comment about it in the code.
![image](https://github.com/ynput/ayon-core/assets/20871534/1483a310-306a-470e-a4c8-a1557bd84e70)

Reply:
how come it was working for @LiborBatek ?

Reply:
> how come it was working for @LiborBatek ?

because tested for ```maya``` I guess...